dev/com.ibm.ws.rest.handler.validator.jca/resources/com/ibm/ws/rest/handler/validator/jca/resources/CWWKOMessages.nlsprops

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.rest.handler.validator.jca/resources/com/ibm/ws/rest/handler/validator/jca/resources/CWWKOMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO Validation REST endpoint
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWKO1560 to CWWKO1569
+
+CWWKO1560_VALIDATION_NOT_IMPLEMENTED=CWWKO1560E: Validation is not provided for connection factory {0}, which implements {1}.
+CWWKO1560_VALIDATION_NOT_IMPLEMENTED.explanation=Validation is only possible for resource adapter connection factories that are JMS connection factories or implement javax.resource.cci.ConnectionFactory or javax.sql.DataSource.
+CWWKO1560_VALIDATION_NOT_IMPLEMENTED.useraction=Do not attempt to validate this type of connection factory.
+
+CWWKO1561_JMS_NOT_ENABLED=CWWKO1561E: JMS feature is not enabled.
+CWWKO1561_JMS_NOT_ENABLED.explanation=The JMS feature is required for this configuration element.
+CWWKO1561_JMS_NOT_ENABLED.useraction=Enable the JMS feature in the server configuration.
+
+CWWKO1562_NO_CONSPEC=CWWKO1562E: Validation cannot be performed with the requested user and password because there is no standard way of locating the proper ConnectionSpec implementation.
```

Comment:
This suggests that validation might be possible with some other user and password (because you said "requested user and password"), consider instead "specified user and password"